README.md

```diff
@@ -28,6 +28,7 @@ You'll also need to clone a new `.env` file from the `.env.tempalate` to store l
 
 ```bash
 $ cp .env.template .env  # (first time only)
+Trello Key and Token has been set in .env to use Trello APIs
```

Comment:
You actually mean TRELLO_KEY and TRELLO_TOKEN.

Also you give no explanation for what they are.

More minor point - this shouldn't be inside the ` ```bash` section. You can preview what the document will look like in VSCode or you can see it when you view the repository in GitHub.